URA-695/reproducibility.py

```
@@ -0,0 +1,181 @@
+    job.wait_on_done()
+    
+    # set the job input
+    bam_file_id = job.describe()['output']['mappings_bam']['$dnanexus_link']
+    job_input = {
+        'gatk_resource_bundle': 
+            dxpy.dxlink('file-F3zvKp84fXX8qJx43zZXP395',
+                      project_id='project-F3zqGV04fXX5j7566869fjFq'),
+        'genome_fastagz': dxpy.dxlink('file-Gb757784XGyY3FPvkPQ74K9z'),
+        'run_bqsr': True,
+        'somatic_algo': 'TNhaplotyper2',
+        'mappings_bam': dxpy.dxlink(bam_file_id)
```

Ruff (0.8.0)

Trailing comma missing

Add trailing comma

(COM812)